_plugins/api.rb

```diff
@@ -85,6 +85,11 @@ def self.generate_locations_endpoint(site)
         'Index of team members by location code', data)
     end
 
+    def self.generate_pages_endpoint(site)
+      # file created through jekyll_pages_api gem, so just need to return information for the index page
```

Comment:
Well, I'm strongly for it, and since it's the existing style in this project... ;-) I'll do it and merge.